linux/deploy_vm/create_unattend_install_iso.yml

```diff
@@ -1,170 +1,154 @@
 # Copyright 2021-2023 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
-- name: "Set fact of the absolute path to unattend install config file template"
+- name: "Set facts of the absolute path to unattend install config file template and installer type"
   ansible.builtin.set_fact:
     unattend_install_template: "{{ main_playbook_path }}/autoinstall/{{ unattend_install_conf }}"
-
-- ansible.builtin.debug:
-    msg:
-      - "The unattend install config file is {{ unattend_install_template }}"
+    unattend_installer: |-
+      {%- if unattend_install_conf is match('Ubuntu/Desktop/Ubiquity') -%}Ubuntu-Ubiquity
+      {%- elif unattend_install_conf is match('(Ubuntu/Server)|(Ubuntu/Desktop/Subiquity)') -%}Ubuntu-Subiquity
+      {%- else -%}{{ unattend_install_conf.split('/')[0] }}
+      {%- endif -%}
+
+- name: "Display unattend install config file template"
+  ansible.builtin.debug:
+    msg: "The unattend install config file template is {{ unattend_install_template }}"
+
+# new_unattend_install_conf is the local file name of unattend install config file created from template
```

Comment:
This comments part seems not in well format or clear.

Reply:
Changed them to ordered list.